content/infrastructure/rippled/installation/build-on-linux-mac-windows.md

```diff
@@ -0,0 +1,19 @@
+---
+html: build-on-linux-mac-windows.md
+parent: install-rippled.html
+blurb: Build rippled on Linux, Mac (macOS), or Windows
+labels:
+  - Core Server
+  - Blockchain
+top_nav_grouping: Popular Pages
+---
+# Build `rippled` on Linux, Mac, or Windows
+
+Building `rippled` across various platforms requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler.
+
+To continue, proceed to [the latest `rippled` build instructions on GitHub](https://github.com/XRPLF/rippled/blob/develop/BUILD.md).
```

Comment:
Shouldn't this still mention the packages?

Reply:
What do you mean? The packages are already documented on https://xrpl.org/install-rippled-on-centos-rhel-with-yum.html and https://xrpl.org/install-rippled-on-ubuntu.html